frontend/src/components/ConfirmationDialog.scss

```
@@ -0,0 +1,56 @@
+.confirmation-dialog-overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.5);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  z-index: 100;
+}
+
+.confirmation-dialog-container {
+  background-color: white;
+  padding: var(--layout-padding-large);
+  border-radius: var(--layout-border-radius-large);
+  box-shadow:
+    0 4px 6px -1px rgba(0, 0, 0, 0.1),
+    0 2px 4px -1px rgba(0, 0, 0, 0.06);
+  max-width: 50%;
+  width: 100%;
+
+  @media (max-width: 767px) {
+    max-width: 80%;
+  }
```

Let's keep the styles all "mobile first" and use media queries to progressively enhance styles as the viewport grows instead of shrinks. We can use the media queries I added recently:

```suggestion
  max-width: 80%;
  width: 100%;

  @include from-md {
    max-width: 50%;
  }
```

The idea is that the baseline styles will work for all screens including the smallest, and the media queries control what happens on larger screens. (the `from-md` mixin is the same as `min-width: 768px`).

This pattern is very common, and mostly we just want to avoid mixing the two patterns in one project because that gets really chaotic 😄